sparse/coo/common.py

```diff
@@ -56,6 +56,7 @@ def linear_loc(coords, shape):
     return out
 
 
+@check_fill_value(2)
```

Comment:
I find it more readable to use keyword argument with literal values, e..g, `@check_zero_fill_value(nargs=2)`